src/com/lits/hw3/hw3Main.java

```diff
@@ -0,0 +1,87 @@
+package com.lits.hw3;
+
+import java.util.Random;
+
+public class hw3Main {
+    public static void main(String[] args) {
+        //У вас є формула : a+11/12(b+b)/12 і є символ,
+        // якшо символ ‘-’ ви ініціалізуєете змінні одними парамеитрами,
+        // якшо ‘+’ то ще іншими і якшо ‘*’ іншими
+        Random r = new Random();
+        var rnd = r.nextInt(2);
+        var symbol = (char)(0);
+        var result = 0;
+        var resultString ="";
+        if (rnd==0) {
```

Comment:
u should check symbols, not numbers

Reply:
dont know how to generate rnd symol from enum ...yet

Reply:
we don't need to generate random numbers, just create char variable, and use it for equal with special symbols